src/main/java/essenmakanan/command/EditRecipeCommand.java

```diff
@@ -40,4 +45,14 @@ public void executeCommand() {
 
     }
 
+    private static List<String> removeWhiteSpaces(String[] splitDetails) {
+        List<String> detailList = new ArrayList();
+        for (String detail : splitDetails) {
+            if (detail.length() > 0) {
+                detailList.add(detail);
```

Comment:
Do you need to use detail.trim() to remove the trailing and leading whitespaces in order to solve #176 ?

Reply:
No, we have already used the white space to split the string, so for example "edit /r bread     n/flavor"
will be split into the array ["bread", "", "", "", "", "", "n/flavor"], I just want to delete this empty string in this case